### PR TITLE
Proofread documentation

### PR DESCRIPTION
**Pull request by bagratte**
Signed-off-by: Bagrat Lazaryan <bagratte@live.com>

### CODE REVIEW

File: themes/helm/assets/sass/docs-content.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+code span {
+  color: black !important;
```
Comment (bacongobbler) [inline themes/helm/assets/sass/docs-content.scss]: Hi! Would you mind explaining the reason behind this change?

Comment (bagratte) [reply themes/helm/assets/sass/docs-content.scss]: I didn't mean to commit this. This was a quick and dirty fix for #499 . Thanks for spotting this. I removed this from the commit.

**Comment by bagratte** [comment]
@bacongobbler Shall we merge this?

**Comment by bridgetkromhout** [comment]
Hi, @bagratte - apologies for the delay. Would you mind resolving the merge conflicts by pulling in changes from upstream? Thanks.

**Comment by bridgetkromhout** [comment]
Hi, @bagratte - Would you mind resolving the merge conflicts by pulling in changes from upstream and pushing them to this branch? As it stands right now, we cannot merge this due to the conflicts. Thanks.

**Pull request merged**